modules/registry/images.yml

```diff
@@ -183,4 +183,11 @@ images:
     tag:
       - "v2.7.0"
     destinations:
-      - registry.sighup.io/fury/goharbor/harbor-exporter
\ No newline at end of file
+      - registry.sighup.io/fury/goharbor/harbor-exporter
+
+  - name: Harbor Trivy Adapter Photon Offline [Fury Kubernetes Registry]
+    source: https://quay.io/repository/sighup/trivy-adapter-photon-offline
```

Comment:
I don't think this source is correct, it should be the url used by docker to pull the image